.github/ISSUE_TEMPLATE/new-diag-study-template.md

```diff
@@ -27,11 +27,11 @@ DOI:
 
 ## Which areas of expertise are particularly relevant to the paper (put an x in the brackets [x])?
 
-- [] virology
-- [] epidemiology
-- [] biostatistics
-- [] immunology
-- [] pharmacology
+- [ ] virology
+- [ ] epidemiology
+- [ ] biostatistics
+- [ ] immunology
+- [ ] pharmacology
 
 <!-- The following questions are the things that we'll want to answer about each paper reporting a diagnostic. If you are ready to answer some of them now, please go ahead and interleave your answers with the prompts. If you are not ready, feel free to go ahead and create the issue now while leaving the prompts blank. -->
```

Comment:
### 
```suggestion
<!-- The following questions are the things that we'll want to answer about each paper reporting a diagnostic. If you are ready to answer some of them now, please create the issue, then copy them into a comment. Otherwise, just create the issue so that a future reviewer can address them in the comments.-->
```

Reply:
Not sure if we actually want this or not?